addons/sales_team/tests/test_sales_team.py

```diff
@@ -78,10 +78,6 @@ def test_default_team_fallback(self):
             team = self.env['crm.team']._get_default_team_id()
             self.assertEqual(team, self.team_responsible)
 
-        self.user_sales_leads.write({
-            'company_ids': [(4, self.company_2.id)],
-            'company_id': self.company_2.id,
-        })
```

Comment:
Not sure why remove this one

Reply:
Because it's redudant with the exact same thing being written below the comment

Reply:
Oh yeah :thinking: